test/e2e/swaps/shared.js

```diff
@@ -160,11 +160,11 @@ const checkNotification = async (driver, options) => {
     '[data-testid="swaps-banner-title"]',
   );
   assert.equal(await boxTitle.getText(), options.title, 'Invalid box title');
```

Comment:
await driver.findElement({
    css: '[data-testid="swaps-banner-title"]',
    text: options.title,
  });

Reply:
I've addressed this comment here e63e33a882